pkg/provider/provider.go

```diff
@@ -195,8 +185,12 @@ func (p *Provider) GetSecretsStoreObjectContent(ctx context.Context, attrib, sec
 	userAssignedIdentityID := strings.TrimSpace(attrib["userAssignedIdentityID"])
 	tenantID := strings.TrimSpace(attrib["tenantId"])
 	cloudEnvFileName := strings.TrimSpace(attrib["cloudEnvFileName"])
-	podName := strings.TrimSpace(attrib["csi.storage.k8s.io/pod.name"])
-	podNamespace := strings.TrimSpace(attrib["csi.storage.k8s.io/pod.namespace"])
+	podName := strings.TrimSpace(attrib[attributePodName])
+	podNamespace := strings.TrimSpace(attrib[attributePodNamespace])
+
+	// attributes for workload identity
+	workloadIdentityClientID := strings.TrimSpace(attrib["clientID"])
```

Comment:
Using `clientID` as the key in `SecretProviderClass` so we can reuse the same key for azure ad applications/managed identity